saos-webapp/src/main/java/pl/edu/icm/saos/webapp/judgment/PageLinkGenerator.java

```diff
@@ -23,19 +25,28 @@
 	 * @param request
 	 * @param pageNoParameterName A page number parameter name. 
 	 */
-	public static String generateSearchPageBaseLink(HttpServletRequest request, String pageNoParameterName) {
+	public static String generateSearchPageBaseLink(HttpServletRequest request, String pageNoParameterName, List<String> filterParameters) {
```

Comment:
filterParameters in a little misleading i think, they are not parameters that will filter something or that are filtered... hmm maybe excludeQueryParameters.